decide/decide.test.cpp

```diff
@@ -1,8 +1,22 @@
 #include <cassert>
+#include <cstdlib>
 #include <decide/decide.hpp>
+#include <decide/lics.hpp>
+#include <iostream>
+#include <sstream>
 
 typedef Connector C;
 
+struct cout_redirect {
+  cout_redirect(std::streambuf *new_buffer)
+      : old(std::cout.rdbuf(new_buffer)) {}
+
+  ~cout_redirect() { std::cout.rdbuf(old); }
+
+private:
+  std::streambuf *old;
+};
```

Comment:
The object will basically redirect the stdout stream to a buffer, until the object is destroyed, then it will reset the stdout stream to the previous buffer (which is stdout in these cases).